src/common/reducers/user.js

```diff
@@ -1,4 +1,16 @@
 // initialised with server side rendered state
-export function user(state = null){
-  return state;
+
+export const DEBUG_SET_USER = 'DEBUG_SET_USER';
+
+export function user(state = null, action) {
+  switch(action.type) {
+    // set individual user properties for debugging
+    case DEBUG_SET_USER:
```

Comment:
It's not.
I wanted it to be clear that it's meant mostly for debugging. Especially that it doesn't reset user data only overrides single properties, so in theory it doesn't really set user to new object, only updates user props.

But maybe `UPDATE_USER` is a good action name for that. So it may serve for debugging and possible use for future.